packages/react-dom/src/__tests__/ReactDOMHooks-test.js

```diff
@@ -133,10 +135,15 @@ describe('ReactDOMHooks', () => {
 
     inputRef.current.value = 'abc';
     inputRef.current.dispatchEvent(
-      new Event('input', {bubbles: true, cancelable: true}),
+      new Event('input', {
+        bubbles: true,
+        cancelable: true,
+      }),
     );
 
-    Scheduler.unstable_flushAll();
+    await act(async () => {
```

Comment:
This should wrap around the input event, not Scheduler. In general we shouldn't ever reference the Scheduler.flush methods unless we're testing some partial rendering implementation detail. Some of our old tests predate `act` but new tests should try to use that as much as possible.

Reply:
Will update, I didn't add the flush here but that's good context for fixing it in the future.